src/styles/style.css

```diff
@@ -0,0 +1,22 @@
+@import url(Blocks/card.css);
+@import url(Blocks/stars.css);
+
+.page {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;

```

Comment:
```suggestion

```

Reply:
We have three fonts in the link for our task. I gave one of styles to the whole page and in places where it differs - changed. Is it wrong?